packages/doc-site/stories/components/anomalyChart/AnomalyChart.mdx

```diff
@@ -60,6 +60,8 @@ includes the following options:
 
 **gestures** (boolean, optional) Specifies whether gestures (pan, zoom and restrict time-span) are enabled. Defaults to `true`
```

Comment:
Defaults to 'true'.
**timeZone** (string, optional): Timezone must be specified as an [IANA timezone string](https://ftp.iana.org/tz/tzdb-2020f/zone1970.tab). If this property is provided, the chart displays all dates in the provided timezone. If not provided, it uses the browser's local timezone.